dll/win32/msv1_0/ntlm/global.c

```diff
@@ -0,0 +1,18 @@
+/*
+ * PROJECT:     Authentication Package DLL
+ * LICENSE:     GPL - See COPYING in the top level directory
+ * FILE:        dll/win32/msv1_0/ntlm/protocol.h
```

Comment:
Please, format the header according to our current style: not `FILE` line and proper license link.
https://reactos.org/wiki/Coding_Style

Reply:
I've fixed all new files and lsa.c, lsa.h. If you wish i can fix the others too.